scopus-import/src/main/java/no/sikt/nva/scopus/ScopusConverter.java

```diff
@@ -336,17 +342,54 @@ private URI extractDOI() {
                                                        .getUri() : null;
     }
 
-    private Optional<TitletextTp> getMainTitleTextTp() {
+    private Optional<TitletextTp> getOriginalMainTitleTextTp() {
         return Optional.ofNullable(extractHead())
                    .map(HeadTp::getCitationTitle)
                    .map(CitationTitleTp::getTitletext)
                    .flatMap(text -> text.stream().filter(this::isTitleOriginal).findFirst());
     }
 
+    /**
+     * Gets the non-original main title for ER (Errata/Corrigendum) citation type.
+     * <p>
+     * ER citation type (mapped to JournalCorrigendum in NVA) does not have an "original"
+     * title type in Scopus, so we use the non-original title as the main title.
```

Comment:
```suggestion
     * Extracts the title for Errata/Corrigendum (Scopus: ER) type. Because this type is
     * a commentary on another document, the concept of "original title" becomes ambiguous,  
     * so the ER type (mapped to JournalCorrigendum in NVA) does not have an "original
     * title" type in Scopus, but rather a "citation title" which we use as the main title.
```